x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/90_loops.yml

```diff
@@ -77,6 +116,16 @@ setup:
              terms:
                field: loose_loop
 
+---
+"loose loop - sort":
+  - do:
+      catch: '/Cyclic dependency detected while resolving runtime fields: loose_loop -> lla -> llb -> llc -> loose_loop/'
+      search:
+        index: sensor
+        body:
+          sort: loose_loop
+---
+"loose loop - docvalue_fields":
```

Comment:
I am aware that splitting tests like this will cause more setup / tear down, yet I think it makes the tests less descriptive and targeted each one at their own specific goal.

Reply:
Makes sense to me. :+1: